src/app/gauge/gauge.component.ts

```diff
@@ -64,12 +64,19 @@ export class GaugeComponent implements OnInit, AfterViewInit, GaugeProps {
   }
 
   ngOnInit(): void {
+    if (!this.start) {
+      this.start = Config.DEF_START;
+    }
+    if (!this.end) {
+      this.end = Config.DEF_END;
+    }
+    this.Configuration=Object.assign(Config,this.config);
```

Comment:
Let's just use the existing `@Input config` for the whole configuration. Introducing a new property is not needed besides it kind of interferes with the SSOT principle. You can do `this.config = Object.assign(Config, this.config)` in the beginning of the `ngOnInit` lifecycle hook then check if `start` and `end` has been modified or not and then update them accordingly, if needed (basically, the comment above).